com.reprezen.swagedit/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,7 +1,315 @@
 eclipse.preferences.version=1
 org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.methodParameters=do not generate
```

Comment:
@ghillairet is there any reason to keep settings in Git? We use Tycho to build it, so I don't see a valid reason  to share the settings, please let me know if you do.